inspectit-oce-core/src/main/java/rocks/inspectit/oce/core/instrumentation/config/InstrumentationConfigurationResolver.java

```diff
@@ -111,6 +111,9 @@ public ClassInstrumentationConfiguration getClassInstrumentationConfiguration(Cl
      */
     public Map<MethodDescription, MethodHookConfiguration> getHookConfigurations(Class<?> clazz) {
         val config = currentConfig;
+        if (isIgnoredClass(clazz, config)) {
```

Comment:
is this anyhow related to this ticket?

Reply:
It's a bug I discovered. We excluded ignored classes from instrumentation but not from hooking.
I saw during testing in the debug output that hooks were built for our own injected classes... :D